index.html

```diff
@@ -62,5 +62,184 @@ <h1>Hi👋, I'm Cephas Tokoli</h1>
          </div>     
         </div>   
     </section>
+    <section class="grid">
```

Comment:
- [ ] Please consider using `display: grid` for the main container of the works/projects card as mentioned in the project requirements 🤓⛏

  > _In order to lay out the elements in "Works" section you should use CSS Grid and Flexbox combined (it is mandatory that you use both). **Use CSS Grid in the main container that contains all of the cards** and use Flexbox to lay out an individual card._